docs/sop.rst

```diff
@@ -0,0 +1,135 @@
+format is presented to the application no matter the envrionment it is running in, enabling a far
+easier development experience.
+
+Operating Apps Managed by Clowder
+---------------------------------
+
+ClowdEnvironment
+++++++++++++++++
+
+**abbreviated to ``env`` in k8s**
+
+The ``ClowdEnvironment`` CRD is responsible for configuring the environment that the Clowder enabled
+apps will interact with. It is a *cluster scoped* CRD and thus must have a unique name inside the
+k8s cluster. For production environments it is usual to have only one ``ClowdEnvironment``, whereas
+in other scenarios &mdash; such as ephemeral testing &mdash; Clowder enables the management of
+multiple environments that operate completely independently from each other.
+
+Providers
+^^^^^^^^^
+
+Clowder provides a number of providers which govern the creation of services, e.g. Kafka topics,
+object storage, etc, that applications may depend on. The ``ClowdEnvironment`` CRD configures these
+providers principally by making use of a provider's mode.
+
+Modes
+^^^^^
+
+Providers often operate in different modes. As an example the Kafka provider can operate in three
+different modes. In *local* mode, the Kafka provider deploys a single node Kafka/Zookeeper instance
+inside the cluster and configures it to auto-create the topics. In *operator* mode, the provider
+assumes a Strimzi Kafka instance is present and will create ``KafkaTopic`` CRs to provide the
+topics.  In *app-interface* mode, no resources are deployed and it is assumed app-interface has
+already created the requested topics. For more information on the configuration of each of these
+providers and their modes, please see the links below.
+
+List of providers:
+
+- `Kafka <https://redhatinsights.github.io/clowder/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-databaseconfig>`_
+- `Logging <https://redhatinsights.github.io/clowder/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-loggingconfig>`_
+- `Object storage <https://redhatinsights.github.io/clowder/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-objectstoreconfig>`_
+- `In-memory DB <https://redhatinsights.github.io/clowder/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-inmemorydbconfig>`_
+- `Relational DB <https://redhatinsights.github.io/clowder/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-databaseconfig>`_
+- `Web <https://redhatinsights.github.io/clowder/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-webconfig>`_
+- `Metrics <https://redhatinsights.github.io/clowder/api_reference.html#k8s-api-cloud-redhat-com-clowder-v2-apis-cloud-redhat-com-v1alpha1-metricsconfig>`_
+
+Target Namespace
+^^^^^^^^^^^^^^^^
+
+Clowder will provision two types of resources during operation, *environmental* and *application*
```

Comment:
I think we can omit the app-specific info here since we're still under the ClowdEnv section.